modules/removeBooks.js

```diff
@@ -0,0 +1,45 @@
+const empty = document.getElementById('empty-id');
+
+function displayEmpty() {
+  empty.classList.toggle('hide');
+}
+
+let store = [];
+
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  removeBook = () => {
+    store = store.filter((book) => book.id !== this.id);
+  };
+}
+
+const addRemoveListener = (book) => {
```

Comment:
Great job @AbdimulhinYussuf3675 🤝

- [ ] Kindly make sure the remove book functionality works as expected on your design. Currently, I am able to add a book but removing a book is not functioning yet.